plugins/europeana/sets.js

```diff
@@ -0,0 +1,65 @@
+import { config } from './';
```

Comment:
Rename this file to plugins/europeana/set.js to be consistent with the others in the same directory.